### ISSUE TITLE
The attention_mask dimension not right？

### ISSUE THREAD

**Issue by zhangyipin** [THUDM/GLM#105]
https://github.com/THUDM/GLM/blob/b8ea51c0f8927bfd3b4497da2b8e6bfa1b9fd6c4/tasks/seq2seq/dataset.py#L856
should output attention mask dimension 3 but 2.
just like
<img width="894" alt="image" src="https://user-images.githubusercontent.com/15236844/222038391-dad27dbb-b74b-4434-b70b-4b82bb0e15a2.png">

**Comment by duzx16** [comment]
Thank you for pointing that out. Already fixed.
